src/Castle.Facilities.WcfIntegration/Service/Discovery/AdHocServiceCatalogDiscovery.cs

```diff
@@ -1,139 +0,0 @@
-// Copyright 2004-2011 Castle Project - http://www.castleproject.org/
-// 
-// Licensed under the Apache License, Version 2.0 (the "License");
-// you may not use this file except in compliance with the License.
-// You may obtain a copy of the License at
-// 
-//     http://www.apache.org/licenses/LICENSE-2.0
-// 
-// Unless required by applicable law or agreed to in writing, software
-// distributed under the License is distributed on an "AS IS" BASIS,
-// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-// See the License for the specific language governing permissions and
-// limitations under the License.
-
-namespace Castle.Facilities.WcfIntegration
-{
-#if DOTNET40
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using System.ServiceModel;
-using System.ServiceModel.Description;
-using System.ServiceModel.Discovery;
-
-	public class AdHocServiceCatalogProbe : AbstractServiceHostAware, IDisposable
```

Comment:
Doesn't the same apply to this class? i.e. DOTNET40 was defined in both the .NET 4.0 and 4.5 configurations, so this was in .NET 4.5.